dask_geopandas/io/arrow.py

```diff
@@ -19,6 +19,7 @@
 from fsspec.core import get_fs_token_paths
 
 DASK_2022_12_0_PLUS = Version(dask.__version__) >= Version("2022.12.0")
+DASK_2023_03_2_PLUS = Version(dask.__version__) >= Version("2023.3.2")
```

Comment:
Once https://github.com/dask/dask/pull/10076/ is merged, I think things will fail in CI until 2023.3.2 is released.

Perhaps we use something like `DASK_2023_03_2_DEV` which is `> Version("2023.3.1")` (strictly greater than)?

Reply:
👍